### PR TITLE
fix: fix test networks display for portfolio view

### LINKED ISSUE #28594: [Bug]: Fiat values displayed for test networks tokens despite having 'Show conversion on test networks' disabled
### Describe the bug

Fiat values displayed for test networks tokens despite having 'Show conversion on test networks' disabled

### Expected behavior

-

### Screenshots/Recordings

https://github.com/user-attachments/assets/c3d8b704-e124-4d71-bb70-4a6c5bab70a7

### Steps to reproduce

1. Switch to a test network
2. Disable 'Show conversion on test networks'

### Error messages or log output

_No response_

### Detection stage

On the development branch

### Version

qa-5 build

### Build type

None

### Browser

Chrome

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

### Severity

_No response_

### PR DESCRIPTION
**Pull request by sahar-fehri**
## **Description**

This PR fixes the display in the asset page and in the main token list when the price checker setting is off

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/28601?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/28594

## **Manual testing steps**

1. Go to this page...
2.
3.

### **After**

https://github.com/user-attachments/assets/a1213b6a-3f23-49f5-be15-a0b369b9016e

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by gambinish — DISMISSED** [review]
Nice, thanks for the recording. LGTM.

Let's also get Salim to look this over since we are trying to sequence our PRs. Let's wait for his approval before merging it 👍

**Review by salimtb — DISMISSED** [review]

**Review by salimtb — APPROVED** [review]

**Review by gambinish — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**
